.eslintrc.yaml

```diff
@@ -0,0 +1,17 @@
+---
+root: true
+parser: "@typescript-eslint/parser"
+plugins:
+  - "@typescript-eslint"
+  - zowe-explorer
+env:
+  node: true
+  es6: true
+rules:
+  zowe-explorer/no-unmocked-core-modules:
+    - error
+    - coreModuleNames:
+        - fs
```

Comment:
this is pretty slick - i didn't know this kind of thing was possible (not so easily anyway) 😄 

should this list contain all node.js core modules used in our project?

Reply:
We use the real functionality of some core modules like `path` at the moment. We would have to go through all the tests and see how to mock them first.

Reply:
gotcha - thanks